src/components/Dropzone/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { ChangeEvent, useRef } from 'react'
+import styled from '@emotion/styled'
+import { color, space, radius, fontWeight } from '../../theme'
+import { H4 } from '../Heading'
+import { SmallText } from '../Text'
+import { BaseButton } from '../BaseButton'
+import ILLUSTRATION from './files.svg'
+
+interface DropzoneProps {
+  title: string
+  subtitle: string
+  action: string
+  ariaLabel?: string
+  accept?: Array<string>
+  onFileChange: Function
```

Comment:
I think it is nice to type arguments and return value as well (if any). So more like this: `onFileChange: (event: SyntaticEvent) => void` (just an example, no idea wht this needs) (in anyway the arrow function definition is nicer that way imo, more explicit)

Reply:
Indeed, I added it as a placeholder and forgot to improve it 🙈 fixed it!